pkg/agent/openflow/client.go

```diff
@@ -470,6 +478,23 @@ func (c *client) deleteFlows(cache *flowCategoryCache, flowCacheKey string) erro
 	return nil
 }
 
+func (c *client) deleteAllFlows(cache *flowCategoryCache) error {
```

Comment:
I feel this method should update the cache like the similar methods do: addFlows, modifyFlows, deleteFlows. Otherwise it's likely people calling the method will forget they need to clean the cache themselves based on their experience with other similar methods.